pay/example/lib/main.dart

```diff
@@ -64,16 +65,23 @@ class _PaySampleAppState extends State<PaySampleApp> {
   @override
   void initState() {
     super.initState();
-    _googlePayConfigFuture =
-        PaymentConfiguration.fromAsset('default_google_pay_config.json');
+    _googlePayConfigFuture = PaymentConfiguration.fromAsset(
+      'default_google_pay_config.json',
+    );
   }
 
   void onGooglePayResult(paymentResult) {
     debugPrint(paymentResult.toString());
   }
 
-  void onApplePayResult(paymentResult) {
+  void onApplePayResult(paymentResult, ApplePaymentConfirmation handler) async {
```

Comment:
I was under the impression that we decided to go with the alternative callback approach (see [comment](https://github.com/google-pay/flutter-plugin/pull/200#discussion_r1623255791) and let me know)

Reply:
ah okay, not sure why i did it with one method, but basically i need to keep the old method and have this new method also, right?

just want to make sure 😁